test/bravo/BravoPostProposalCheck.sol

```diff
@@ -0,0 +1,80 @@
+pragma solidity ^0.8.0;
+
+import "@forge-std/Test.sol";
+
+import {GovernorBravoDelegator} from "@comp-governance/GovernorBravoDelegator.sol";
+
+import {Addresses} from "@forge-proposal-simulator/addresses/Addresses.sol";
+import {MockERC20Votes} from "@forge-proposal-simulator/mocks/MockERC20Votes.sol";
+import {Timelock} from "@forge-proposal-simulator/mocks/bravo/Timelock.sol";
+import {GovernorBravoDelegate} from "@forge-proposal-simulator/mocks/bravo/GovernorBravoDelegate.sol";
+
+import {MockBravoProposal} from "proposals/MockBravoProposal.sol";
+
+// @notice this is a helper contract to execute proposals before running integration tests.
+// @dev should be inherited by integration test contracts.
+contract BravoPostProposalCheck is Test {
+    Addresses public addresses;
+
+    function setUp() public {
+        addresses = new Addresses("./addresses/Addresses.json");
+        vm.makePersistent(address(addresses));
+
+        MockBravoProposal bravoProposal = new MockBravoProposal();
+
+        // Set the addresses contract
+        bravoProposal.setAddresses(addresses);
+
+        // Verify if the governor address is a contract; if is not (e.g. running on a empty blockchain node), deploy a new governor and update the address.
+        address governor = addresses.getAddress("PROTOCOL_GOVERNOR");
+        uint256 governorSize;
+        assembly {
```

Comment:
I guess it makes more sense not to have to etch the governor contract here and run in a testnet fork instead, as we do (currently) on the FPS repo. I believe the only mocks we should have in this repo are the ones to exemplify the FPS functions, like having a Vault/Token to example the deploy function. We can then create CI examples to generate calldata/run integration tests that run against the testnet, similar to the ones we have on Moonwell. What do you think, @ElliotFriedman?